src/Raven.Studio/typescript/viewmodels/resources/serverDashboard.ts

```diff
@@ -413,22 +415,39 @@ class serverDashboard extends viewModelBase {
             const diff = now.diff(startTime);
             return generalUtils.formatDuration(moment.duration(diff), true, 2);
         });
-        
+
         this.formattedStartTime = ko.pureComputed(() => {
             const start = this.startUpTime();
             const dateFormat = "YYYY MMMM Do, h:mm A";
             return start ? start.local().format(dateFormat) : "";
-        })
+        });
+
+        this.node = ko.pureComputed(() => {
+            const topology = this.clusterManager.topology();
+            const nodeTag = topology.nodeTag();
+            const node = topology.nodes().filter(x => x.tag() === nodeTag)[0];
```

Comment:
return topology.nodes().find(x => x.tag() === nodeTag);
and remove if (!node) {
 +                return null;
 +            }
 +
 +            return node;

Reply:
done